test/deadline_adaptor_for_cinema4d/unit/Cinema4DAdaptor/test_adaptor.py

```diff
@@ -24,36 +23,36 @@ def init_data() -> dict:
 
 class TestCinema4DAdaptor_on_cleanup:
 
-    def test_handle_critical(self, init_data: dict) -> None:
-        """Tests that the _handle_error method throws a critical runtime error correctly"""
-        # GIVEN
-        adaptor = Cinema4DAdaptor(init_data)
-
-        # WHEN
-        error_regex = re.compile(".*Error: .*|.*\\[Error\\].*|.*CRITICAL: .*")
-        stdout = "CRITICAL: Stop [ge_file.cpp(1172)]"
-
-        match = error_regex.search(stdout)
-        if match:
-            adaptor._handle_error(match)
-
-        # THEN
-        assert match is not None
-        assert str(adaptor._exc_info) == f"Cinema4D Encountered an Error: {stdout}"
-
-    def test_handle_error(self, init_data: dict) -> None:
+    @pytest.mark.parametrize(
+        "stdout,error_expected",
+        [
+            # Only critical stop errors should fail the job
+            ("CRITICAL: Stop [ge_file.cpp(1172)]", True),
+            # Any errors should fail the job
+            ("Redshift Error: Maxon licensing error: User not logged in (7)", True),
```

Comment:
nit: should we also add an example for the `.*\\[Error\\].*` regex?

Reply:
Sure, I can add it in a follow up PR.